### ISSUE TITLE
Wrong GPU device usage

### ISSUE THREAD

**Issue by ngraham20** [flightlessmango/MangoHud#1293]
**Describe the bug**

The GPU usage % that is displayed is of the wrong device. I have a dGPU and an iGPU, and while the gpu_name shows the dedicated GPU, the percentage is of the iGPU.

**List relevant hardware/software information**
- Linux Distribution: Nobara 39
- MangoHud version: v0.7.1
- GPU: AMD Radeon 7700S

**To Reproduce**
Steps to reproduce the behavior: Run Mangohud

**Expected behavior**

**Screenshots**
The percentage MangoHUD shows against AMDGPU top. The Graphics card shown is labeled next to `Info`
![image](https://github.com/flightlessmango/MangoHud/assets/22180760/0ed6ab1e-18af-4baa-8cc1-6120e39dc6bc)
![image](https://github.com/flightlessmango/MangoHud/assets/22180760/3b8111f3-4364-4613-bcbd-bcbb00fd56fe)

**Comment by Lassebq** [comment]
Have you already tried using `pci_dev` config option?

**Comment by emansom** [comment]
Also experiencing this on a RX 7600 (CPU+iGPU) + RX 580 (GPU) combo. Weirdly enough it only seems to trigger in one game of mine: Need for Speed Most Wanted 2012, where as in GTA 5 it picks up the RX 580 fine.

All Lutris settings across both games matched, no PRIME enabled.

**Comment by emansom** [comment]
`nvtop` has some sort mechanism to determine which application belongs to which GPU, could this be leveraged in MangoHud to identify which GPU is rendering the game?

**Comment by flightlessmango** [comment]
Yes it is possible to figure out which GPU is being used using said method. I'm leaning towards implementing that or just going ahead and support multiple GPUs

**Comment by ngraham20** [comment]
> Have you already tried using `pci_dev` config option?

At this point, I can't remember, and using goverlay has prevented the issue, so I don't know :sweat_smile:

**Comment by joshw34** [comment]
I had the same problem. The only way i could fix was to disable my igpu in bios. Not a perfect solution, but it worked

**Comment by ngraham20** [comment]
That seems a bit drastic. On devices with a mux, the igpu is used for applications that don't need the power, reducing battery drain and power consumption

**Comment by flightlessmango** [comment]
This should be resolved now that multiple GPUs are supported
